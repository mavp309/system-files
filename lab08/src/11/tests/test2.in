Sneha
19
44556677
90
