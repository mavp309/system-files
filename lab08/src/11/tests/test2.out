Name: Sneha
Age: 19
Roll: 44656677
Marks: 90
